### PR TITLE
0.9.3rc

### PR DESCRIPTION
**Pull request by arokem**
As a stop-gap, we might want to release a 0.9.3, so that we can provide our users on debian platform with something that works through apt-get.

### CODE REVIEW

**Comment by yarikoptic** [comment]
not that travis is really unhappy

File: dipy/denoise/tests/test_noise_estimate.py [hunk 1]
```diff
@@ -29,6 +32,26 @@ def test_piesno():
...
+    # Test using the median as the initial estimation
+    initial_estimation = (np.median(sigma) /
+                          np.sqrt(2 * _inv_nchi_cdf(1, 1, 0.5)))
+    sigma, mask = _piesno_3D(rician_noise, N=1, alpha=0.01, l=1, eps=1e-10,
+                             return_mask=True,
+                             initial_estimation=initial_estimation)
+
+    assert_(np.abs(sigma - 50) / sigma < 0.03)
+
+>>>>>>> 1868e71... BF: Cannot coerce into bool (?).
```
Comment (samuelstjean) [inline dipy/denoise/tests/test_noise_estimate.py]: poor tests had a merge problem

Comment (arokem) [inline dipy/denoise/tests/test_noise_estimate.py]: Oh bugger

**Comment by arokem** [comment]
I think that given the Travis issues, we'll have to ditch this plan. I just don't have time to look into this in the next few days, so it seems like going with 0.10.1 is the best way forward. @yarikoptic - I will create the documentation folder for you from that -- hopefully some time today.

**Pull request closed without merging**